frontend/packages/dev-console/src/components/topology/Topology.tsx

```diff
@@ -27,6 +27,7 @@ import { topologyModelFromDataModel } from './topology-utils';
 import { layoutFactory, COLA_LAYOUT, COLA_FORCE_LAYOUT } from './layouts/layoutFactory';
 import ComponentFactory from './componentFactory';
 import { TYPE_APPLICATION_GROUP } from './const';
+import TopologyFilterBar from './filters/TopologyFilterBar';
```

Comment:
This throws an error for me.
the TopologyFilterBar is located at '../topology2/filters/TopologyFilterBar'

Reply:
@nemesis09 fixed all the import errors